### PR TITLE
Remove unnecessary default formatting in our examples

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: examples/compiled/area.vg.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
           "orient": {"value": "vertical"},
           "fill": {"value": "#4c78a8"},
           "tooltip": {
-            "signal": "{\"date (year-month)\": timeFormat(datum[\"yearmonth_date\"], '%Y'), \"count\": format(datum[\"sum_count\"], \"\")}"
+            "signal": "{\"date (year-month)\": timeFormat(datum[\"yearmonth_date\"], '%b %Y'), \"count\": format(datum[\"sum_count\"], \"\")}"
```
[the remark this answers is not in the document]
Comment (kanitw) [reply examples/compiled/area.vg.json]: Probably yes.

**Comment by kanitw** [comment]
@domoritz  this is failing because the master is failing lint. lol

File: examples/compiled/area.vg.json [hunk 2]
```diff
@@ -77,7 +77,7 @@
       "tickCount": {"signal": "ceil(width/40)"},
       "encode": {
         "labels": {
-          "update": {"text": {"signal": "timeFormat(datum.value, '%Y')"}}
+          "update": {"text": {"signal": "timeFormat(datum.value, '%b %Y')"}}
```
Comment (domoritz) [inline examples/compiled/area.vg.json]: The old value was much better here, no?

Comment (kanitw) [reply examples/compiled/area.vg.json]: ok i'll undo

**Comment by kanitw** [comment]
I've fixed this to have no SVG changes, so I'll merge this.

**Pull request merged**